src/sentry/issues/mark_reviewed.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from typing import Any, Dict, List
+
+from sentry.models import Group, Project, User
+from sentry.models.groupinbox import (
+    GroupInboxReason,
+    GroupInboxRemoveAction,
+    add_group_to_inbox,
+    remove_group_from_inbox,
+)
+from sentry.signals import issue_mark_reviewed
+
+
+def mark_reviewed(
+    in_inbox: bool,
```

Comment:
This name feels a little misleading to me, since at the time you pass the value in, it's actually not yet true. I'm trying to think of a better alternative, without resorting to something ridiculous like `requested_move_direction_is_in_rather_than_out`. Maybe `should_move_in`? That doesn't really encapsulate the fact that `False` means "move out" rather than "do nothing," but I'm failing to think of something better at the moment. WDYT?

Reply:
> requested_move_direction_is_in_rather_than_out  

LOL 

I went with `mark_reviewed` since that's the feature name, but I see your point. Maybe `update_inbox`? We're going to be removing this feature in issue states so it won't be around for much longer.